README.md

```diff
@@ -15,14 +15,22 @@
 
 ### Compiling from Source
 
-If you prefer to compile from source or are contributing to the project, you can install `kuadrantctl` using `go install`. This method requires Golang 1.21 or newer.
+If you prefer to compile from source or are contributing to the project, you can install `kuadrantctl` using `go install` or `make install`. This method requires Golang 1.21 or newer.
 
 ```bash
-go install github.com/kuadrant/kuadrantctl@latest
+VERSION=$(git rev-parse --short=7 HEAD) go install -ldflags='-X "github.com/kuadrant/kuadrantctl/version.Version=dev - '$VERSION'"' 'github.com/kuadrant/kuadrantctl'
```

Comment:
hmmm that bulkier then i thought, im thinking we just go with the make install target much neater and easier to read

Reply:
Sure, sounds good. :)